### PR TITLE
DM-33661: narrativelog: add support

### PR DESCRIPTION
**Pull request by r-owen**
including configuration for summit, base, and Tucson test stand.

Also tweak a few exposurelog settings.

### CODE REVIEW

File: setup.cfg
Comment (athornton) [inline setup.cfg:1]: Pretty sure we use line-length 79; not sure we even use flake8.  Take a look at the settings for https://github.com/lsst-sqre/mobu (in pyproject.toml)--I think those are canonical.

**Review by athornton — CHANGES_REQUESTED** [review]
Bring the black/flake8 settings into line with other SQuaRE projects and I think we're good.  The narrativelog stuff looks fine, and I'm glad the linter caught useless f-strings and unused imports and stuff.

File: setup.cfg
Comment (jonathansick) [reply setup.cfg]: Our "standard" configurations aren't in mobu — they're in the template: https://github.com/lsst/templates/tree/main/project_templates/fastapi_safir_app/example

Mobu has a few extra configs for its specific use cases.

**Comment by r-owen** [comment]
I have moved the reformat to a different PR that I'll assign to @jonathansick once this one is merged and I can rebase it. So now the only changes are due to narrativelog. Please have another look.

**Comment by r-owen** [comment]
As to the failed github action...I'm wondering if this needs the actual narrativelog service to pass. That's waiting for review so has not yet been tagged...

**Review by athornton — APPROVED** [review]
LGTM as long as you're sure that changing site_id isn't going to break roe.

**Comment by jonathansick** [comment]
We've got to finish https://github.com/lsst-sqre/charts/pull/646 first and publish that chart so the helm CI test can try to deploy narrativelog in minikube and validate that the chart works.

File: services/exposurelog/values-roe.yaml
Comment (r-owen) [reply services/exposurelog/values-roe.yaml:4]: Good question. This deployment is a sandbox (it is using the built-in test registries) and so it should use site_id=test. That said, it's not a big deal. It only controls the site_id field for new messages.

File: setup.cfg
Comment (r-owen) [reply setup.cfg]: I pulled it out. I hope somebody will fix this because it's a nuisance to edit files when your editor is set to reformat them and there is no format configuration in the package.

**Pull request merged**
